pilot/pkg/features/pilot.go

```diff
@@ -689,6 +689,12 @@ var (
 		false,
 		"If set, it allows creating inbound listeners for service ports and sidecar ingress listeners ",
 	).Get()
+
+	EnableNativeStats = env.Register(
+		"TELEMETRY_USE_NATIVE_STATS",
+		true,
```

Comment:
Do we want this on by default or off? I don't have enough context to form an opinion myself, but i thought you had previously said it needed a lot of validation before production usage?

Reply:
The answer depends on whether we're doing stability tests? If not, then we should default disable it.

Reply:
Not sure the current state, but whatever we have is surface level at best (non-existent at worst)

Reply:
Telemetry API is opt-in already though. I haven't touched the default installation profile.

Reply:
I am not sure why it matters that it is opt-in; users opted into an API not an experimental runtime change.

If we don't think (or if we don't know yet) if its safe, it should be off by default probably.

Reply:
The crux of the issue is the lack of release qualification testing. Will disable it, run tests manually on release build, and then maybe turn it back on.